clipper_admin/clipper_manager.py

```diff
@@ -58,20 +58,32 @@
 EXTERNALLY_MANAGED_MODEL = "EXTERNAL"
 
 
+from numpy import *
+import scipy as sp
+import warnings
+warnings.filterwarnings("ignore", category=FutureWarning)
+from pandas import *
+import pandas.rpy.common as com
+from rpy2.robjects.packages import importr
+import rpy2.robjects as ro
+from rpy2.robjects.packages import importr
+stats = importr('stats')
```

Comment:
Import the R specific imports at the beginning of the call to `deploy_R_model`. See the [Spark method](https://github.com/ucbrise/clipper/blob/develop/clipper_admin/clipper_manager.py#L631-L639) as an example.